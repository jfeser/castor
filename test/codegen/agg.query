select([(1.0 + 2.0) as x, avg(a) as y, count() as z, sum(a) as r, min(a) as s, max(a) as t],
  alist(r2, ascalar(0.a)))