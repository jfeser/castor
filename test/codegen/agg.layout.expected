0:10 List body
0:2 Scalar (=(Fixed ((value -42) (scale 100))))
0:0 List len (=10)
0:0 List count (=5)
2:2 Scalar (=(Fixed ((value 1) (scale 100))))
4:2 Scalar (=(Fixed ((value 88) (scale 100))))
6:2 Scalar (=(Fixed ((value 5) (scale 1))))
8:2 Scalar (=(Fixed ((value 3442) (scale 100))))
